sharding-sql-test/src/main/java/org/apache/shardingsphere/test/sql/loader/SQLCasesLoader.java

```diff
@@ -73,13 +75,14 @@ public SQLCasesLoader(final String rootDirection) {
         return result;
     }
     
+    @SneakyThrows
     private static Map<String, SQLCase> loadSQLCasesFromTargetDirectory(final String path) {
         Map<String, SQLCase> result = new TreeMap<>();
         URL url = SQLCasesLoader.class.getClassLoader().getResource(path);
         if (null == url) {
             return result;
         }
-        File filePath = new File(url.getPath());
+        File filePath = new File(URLDecoder.decode(url.getPath(), "UTF-8"));
```

Comment:
How about if the url is not for UTF-8 encoding? Use default encoding may better.

Reply:
URLDecoder.decode() Without loss of performance, but also to prevent Chinese garbled code，

Reply:
How about if the url is not for UTF-8 encoding?

Reply:
url.getPath() is string, if its charset is wrong, decode is unuseful.